fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -607,9 +608,11 @@ public Response updateSparql(final InputStream requestBodyStream)
     public Response createObject(@HeaderParam(CONTENT_DISPOSITION) final ContentDisposition contentDisposition,
                                  @HeaderParam(CONTENT_TYPE) final MediaType requestContentType,
                                  @HeaderParam("Slug") final String slug,
-            final InputStream requestBodyStream,
+        final InputStream requestBodyStream,
                                  @HeaderParam(LINK) final List<String> rawLinks,
-                                 @HeaderParam("Digest") final String digest)
+                                 @HeaderParam("Digest") final String digest,
+                                 @HeaderParam(ATOMIC_START) final String AtomicStart,
```

Comment:
I believe the detection of the Atomic-Id, Atomic-Start, and Atomic-Commit will all be handled at a higher level in the stack.  You should be able to access the current Transaction by calling FedoraBaseResource.session().getTransaction().  I have a PR in now that will add FedoraBaseResource.getTransaction() that effectively will do the same thing.  So all code related to resolving the transaction can go away in FedoraLdp.

Reply:
This is the Jersey endpoint that receives the request, no? Perhaps my understanding of JAX-RS is wrong but I thought this is where the request originates? Are you writing some sort of Interceptor?

Reply:
This controller:
https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-http-api/src/main/java/org/fcrepo/http/api/Transactions.java#L48
has been used for starting and committing transactions in the past, so I suspect that is where it'll end up going forward. This is based off of the assumption that we be following Peter's current transaction API suggestions, which involves making a POST to `/rest/fcr:tx` to start the transaction:
https://github.com/fcrepo/fcrepo-specification-atomic-operations/pull/8/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR120

So no header is needed to start a transaction. Only the Atomic-ID header would make it to FedoraLdp. Some of my earlier documentation from the sprint was using Atomic-Start based off of older versions of the spec before it was updated. Sorry for the confusion.

Reply:
Ohhhhhh okay. So I should remove the `Atomic-Start` header. Yes, I see that it has been removed, that was a mistake I was probably seeing the removed content of the atomic-batch-operations PR and still registering it.